.github/workflows/techlabblog-deploy-dev.yml

```diff
@@ -0,0 +1,50 @@
+name: Techlab Blog | Deploy Dev
+
+on:
+  push:
+    branches: [ft/techlabblog-workflow]
+    paths:
+      - "apps/techlabblog/**"
+      - "Dockerfile"
+      - ".github/workflows/techlabblog-deploy-dev.yml"
+
+# This allows a subsequently queued workflow run to interrupt previous runs
+concurrency:
+  group: "${{ github.workflow }} @ ${{ github.ref }}"
+  cancel-in-progress: true
+
+env:
+  SENTRY_ENVIRONMENT: "development"
+
+jobs:
+  env-setup:
+    name: Environment Setup
+    runs-on: ubuntu-latest
+    outputs:
+      SENTRY_ENVIRONMENT: ${{ steps.env-setup.outputs.SENTRY_ENVIRONMENT }}
+    steps:
+      - name: Environment Setup
+        id: env-setup
+        run: |
+          echo "SENTRY_ENVIRONMENT=${{ env.SENTRY_ENVIRONMENT }}" >> $GITHUB_ENV
```

Comment:
Why do we (still/even) need this? Can't we use `env.SENTRY_ENVIRONMENT` directly wherever it's needed in this workflow?